pubspec.yaml

```
@@ -14,7 +14,7 @@ dependencies:
   convert: ^3.0.2
   crypto: ^3.0.1
   frontend_server_client: ^3.2.0
-  http: ^0.13.3
+  http: ">=0.13.3 <2.0.0"
```

I think we can just go straight to 1.0.0.

In the sdk we'll anyway just use whatever it provides.
```suggestion
  http: "^1.0.0"
```

---

If we want to do that we'll also need to update transitive deps.

https://github.com/google/dart-neats/pull/209